tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -132,6 +132,11 @@ namespace tuplex {
         auto tstage = dynamic_cast<TransformStage*>(stage);
         if(tstage)
             executeTransformStage(tstage);
+        else if (dynamic_cast<SortStage*>(stage)) {
+            auto stg = dynamic_cast<SortStage*>(stage);
+            stg->setResultSet(stage->resultSet());
```

Comment:
maybe try to remove duplicate dynamic_cast mode (can you use directly check on stage?)